src/sp_var_value.c

```diff
@@ -172,7 +172,6 @@ static zval *get_unknown_type(const char *restrict value, zval *zvalue,
                               const sp_tree *const tree, bool is_param) {
   if (ce) {
     zvalue = get_entry_hashtable(&ce->constants_table, value, strlen(value));
-    ce = NULL;
```

Comment:
Did you check if `ce` isn't used in the callstack?

Reply:
`ce` is a local variable to this function, so an assignment here has no effect in any caller.

Reply:
`ce` is passed as a parameter to the function, it's not a local variable, is it?

Reply:
`ce` is a local copy of the address, so assigning `NULL` here has no effect to the address in the variable in the caller.